enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,244 @@
+As Docker service is handled by the user, it is up to the user to decide if Docker should continue to run or not.
+When it comes to MachineSet, the user can choose a VM image that doesn't have Docker. Containerd will be the only runtime.
+If the MachineSet uses a VM image that comes with Docker, containerd will run along with Docker service. But Kubernetes will
+use only containerd. Customers can stop the Docker service if it is no longer needed. Even if Docker service gets started at any time,
+it can co-exist with containerd.
+
+Steps to install containerd as service:
+- scp containerd/related executables into Windows VM
+- copy the files into `C:\k\bin` folder
+- create containerd config file
+- register containerd as service
+- start containerd as a Windows service
+
+## Network changes
+Current CNI/IPAM will be used for containerd and no change will be made in HNS-Network and HNS-Endpoint creation
+steps. The config file which will be used by containerd will point to the same CNI/IPAM executables.
+
+## WMCO CLI option
+WMCO CLI option 'runtime' will be used to make containerd as the default runtime.  In the 4.10 community operator, it
+will be specified as `--runtime containerd`. In the absence of this flag, Docker will become the default runtime. We remove
+this flag in 4.11 and make containerd the default runtime. At any given point in time, we have one runtime support and
+do not allow switching between runtime.
+
+## Containerd logging
+Containerd can be started with parameters in which we can enable logging and specify the file path to
+log warnings/errors. Log files will be stored at c:\var\log\containerd. Pod logs will be stored in the
+same location (C:\var\log\pods)
+
+## Upgrade
+WMCO supports upgrade. Please refer to the [WMCO-Upgrade](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator-upgrades.md)
+enhancement proposal document to learn more about WMCO upgrade.
+
+The procedure for an upgrade is as follows:
+1) MachineSet
+   * As part of upgrade Machine object gets deleted, resulting in the drain and deletion of the Windows node.
+   * New Windows VM will be created.
+   * The upgraded WMCO instance will configure Windows VM with containerd as the default runtime.
+   * Once the Windows node joins the Kubernetes cluster, the user can deploy pods on the Windows node.
+2) BYOH ( Bring Your Own Host)
+   * upgrade process will uninstall kubelet, kube-proxy, CNI, and hybrid-overlay components that were installed by WMCO.
+   * Windows OS-specific configurations like HNS-Network that were created by WMCO instance would be deleted.
+   * Once cleanup is done, WMCO instance will install containerd along with kubelet, kube-proxy, CNI and
+     hybrid-overlay.
+   * Kubelet will start as a service. Containerd will be the default runtime.
+   * Once the Windows node joins the Kubernetes cluster, the user can deploy pods on the Windows node.
+
```

Comment:
Add a note here saying that the if the Docker service is present, it will continue to run. I know you have mentioned this before but a call out is needed given you are talking about details here.